runelite-api/src/main/java/net/runelite/api/AnimationID.java

```diff
@@ -103,6 +103,10 @@
 	public static final int FISHING_CRUSHING_INFERNAL_EELS = 7553;
 	public static final int FISHING_CUTTING_SACRED_EELS = 7151;
 	public static final int FISHING_BAREHAND = 6709;
+	public static final int FISHING_BAREHAND_WINDUP_1 = 6703;
+	public static final int FISHING_BAREHAND_WINDUP_2 = 6704;
+	public static final int FISHING_BAREHAND_CAUGHT_1 = 6710;
+	public static final int FISHING_BAREHAND_CAUGHT_2 = 6711;
```

Comment:
These are not named correctly. Since these apply only to tuna, they should be named appropriately to reflect that.
```suggestion
	public static final int FISHING_BAREHAND_CAUGHT_TUNA_1 = 6710;
	public static final int FISHING_BAREHAND_CAUGHT_TUNA_2 = 6711;
```

Reply:
Thank you for finding the rest of these animations.